CMakeLists.txt

```diff
@@ -305,6 +305,14 @@ IF(WITH_CORE)
     # (Qt 3D was introduced in 5.7 but it is not stable enough in that branch)
     # Qt 5.8 is missing some classes, https://github.com/qgis/QGIS/pull/5203#discussion_r142319862
     SET(QT_MIN_VERSION 5.9.0)
+    # on recent Debian and Ubuntu the headers from Qt3DExtras library have been removed
```

Comment:
Out of curiosity, what is there a reason for this removal?

Reply:
In recent Qt versions that lib is marked as unstable (tech preview?) so the maintainers removed headers but kept the lib binary. I do not fully understand that logic.

See https://lists.osgeo.org/pipermail/qgis-developer/2018-April/052925.html

Reply:
:smile: sounds a bit like the pyqt-webkit decision